### ISSUE TITLE
Password error when creating a new account

### ISSUE THREAD

**Issue by Marcioghiraldelli** [AzuraCast/AzuraCast#6292]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

xxx

### What happened?

I'm using the latest version of Azuracast Roling release and I realize that every new account created, even if I put in the correct email and password, I still can't verify the password or access data.
It only works if I go to the terminal and run ./docker.sh cli azuracast:account:reset-password

Then it generates a new password for the new email that I just registered. But the password that I enter myself does not work and the password that azuracast creates does not let me change it giving the following error

InvalidArgumentException at /var/azuracast/www/src/Controller/Api/Frontend/Account/PutPasswordAction.php L31: Invalid current password.

In short, when creating a new account, the password used at the beginning does not work and does not let the panel change it.

### Relevant log output

```shell
InvalidArgumentException at /var/azuracast/www/src/Controller/Api/Frontend/Account/PutPasswordAction.php L31: Invalid current password.
```

**Comment by Chrisx84** [comment]
Im having the same issue with 0.18.3

**Comment by BusterNeece** [comment]
Thank you for the report! I identified the cause of this and have fixed it in the latest Rolling Release. We're still working on fine-tuning things with Liquidsoap 2.2.x, but once that's worked out there will be a new stable release including it.
